client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Properties panel quality of life improvements</h4>
-          We removed automatic (magic) sorting from all properties panels. Additional minor improvements ensure that implementing Camunda 7 and 8 is simpler than ever.
+          <h4>Enhanced Form Component Documentation</h4>
```

Comment:
```suggestion
          <h4>Enhanced form components documentation</h4>
```

Reply:
addressed in [049a158](https://github.com/camunda/camunda-modeler/pull/4384/commits/049a1588313c7b31edbecb51d039ecb4e7b4132a)